### PR TITLE
Fix test Category suite full

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x 
| Description?  | Fix test Category Suite full (fix selectors and add wait for elements before next action)
| Type?         |  refacto 
| Category?     | TE
| BC breaks?    | Does it break backward compatibility? no
| Deprecations? | Does it deprecate an existing feature? no
| Fixed ticket? | no
| How to test?  | TEST_PATH=full/03_category/1_create_edit_delete_category npm run specific-test -- --URL=http://localhost/shopName

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13493)

### CODE REVIEW

**Comment by matks** [comment]
Travis failed for bad reasons, I restart it

File: tests/E2E/test/campaigns/common_scenarios/category.js [hunk 1]
```diff
@@ -48,9 +48,10 @@ module.exports = {
       test('should click on "Save" button', () => {
         return promise
           .then(() => client.scrollWaitForExistAndClick(CategorySubMenu.save_button, 50))
-          .then(() => client.getTextInVar(CategorySubMenu.category_number_span, "number_category"));
+          .then(() => client.getTextInVar(CategorySubMenu.category_number_span, "number_category"))
+          .then(() => tab['number_category'] = tab['number_category'].split('(')[1].split(')')[0]);
```
Comment (matks) [inline tests/E2E/test/campaigns/common_scenarios/category.js:52]: Not the best way to select this item but I guess you had nothing easier to use.

For future puppeeter tests, when you have such complex usecases, let's work together in order to to provide you easier items to use 😉

**Review by mbadrani — DISMISSED** [review]
Thanks Boubker for your changes

**Review by mbadrani — DISMISSED** [review]
Tested together thanks

**Pull request closed without merging**